deployment/base/maas-api/networking/httproute.yaml

```diff
@@ -8,6 +8,17 @@ spec:
     - name: openshift-ai-inference
       namespace: openshift-ingress
   rules:
+    # OpenAI-compatible /v1/models endpoint at top level
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /v1/models
+      backendRefs:
+        - name: maas-api
+          namespace: platform-services
```

Comment:
```suggestion
```

The route is in the same namespace, no need to be explicit. And also this ns does not exist anymore :)